api4/config.go

```diff
@@ -320,6 +323,11 @@ func makeFilterConfigByPermission(accessType filterType) func(c *Context, struct
 			if tagValue == model.ConfigAccessTagCloudRestrictable {
 				continue
 			}
+			// The 'access' tag 'any' checks for any SYSCONSOLE read permission and grants access if any read permission is allowed.
+			if tagValue == AnyTagValue && c.App.SessionHasPermissionToAny(*c.App.Session(), model.SysconsoleReadPermissions){
+				return true
+			}
+
```

Comment:
All of the other `access` tags apply to both read and write. Is this `any` value only for reading?